README.md

```diff
@@ -1,2 +1,162 @@
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Add new students or teachers**
+- **Add new books**
+- **Save records of who borrowed a given book and when**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- GitHub account;
+- Git installed on your OS;
+- VSCode (or another code editor);
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/) installed;
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Login to your GitHub account. Clone this repository to your desired folder:
+
+> cd my-folder
+> git clone git@github.com:MostafaShahat2021/OOP-school-library.git
+
+### Install
+
+Install this project with:
+(will be added in following projects)
+
+### Usage
+
+To run the project, execute the following command:
+(will be added in following projects)
+
+### Run tests
+
+To run tests, run the following command:
+(will be added in following projects)
+
+You can also check linter errors by running these commands:
+> rubocop
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤**Mostafa Shahat**
+
+- GitHub: [@MostafaShahat2021](https://github.com/MostafaShahat2021)
+- LinkedIn: [@mostafa-shahat](https://www.linkedin.com/in/mostafa-shahat/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **use the "Decorator" design pattern**
+- [ ] **set up associations**
+- [ ] **add basic UI**
+- [ ] **preserve data**
+- [ ] **add unit tests**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just star it!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse program for the knowledge and skills I have acquired in Modules 1-3.
+
+I am grateful to all previous project reviewers for their advice. Thanks to them I could build this app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Why is there no Live Demo link?**
+
+  - This is a student project and the Live Demo will be added after reaching the corresponding phase of the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Great job working on this milestone and updating the ReadMe. Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.
![Screen Shot 2023-11-09 at 11 52 34 AM](https://github.com/MostafaShahat2021/OOP-school-library/assets/62849872/94798157-2312-49ea-b772-51d925c6327e)

Reply:
Hey @t-yanick, Thanks for your suggestion, I'll fix that 😄